WordPress/Sniffs/WhiteSpace/ControlStructureSpacingSniff.php

```diff
@@ -506,6 +507,16 @@ public function process_token( $stackPtr ) {
 			return;
 		}
 
+		if ( T_COMMENT === $this->tokens[ $trailingContent ]['code'] ) {
+			if ( $this->tokens[ $trailingContent ]['line'] === $this->tokens[ $scopeCloser ]['line'] ) {
+				if ( '//end' === substr( $this->tokens[ $trailingContent ]['content'], 0, 5 ) ) {
```

Comment:
What about `// end`, `// End`, etc. ? Does it really even matter what the comment looks like here? Would be good to see some tests that show what this part does and that it works correctly.